policyengine_us/variables/gov/states/mt/tax/income/exemptions/mt_dependent_exemptions.py

```diff
@@ -1,23 +1,20 @@
 from policyengine_us.model_api import *
 
 
-class mt_dependent_exemption_count(Variable):
-    value_type = float
-    entity = TaxUnit
-    label = "Number of Montana dependent exemptions"
+class mt_dependent_exemptions(Variable):
+    value_type = int
+    entity = Person
+    label = "Montana number of dependent exemption for each dependent"
     unit = USD
     definition_period = YEAR
     reference = "https://regulations.justia.com/states/montana/department-42/chapter-42-15/subchapter-42-15-4/rule-42-15-403/"
     defined_for = StateCode.MT
 
-    def formula(tax_unit, period, parameters):
-        person = tax_unit.members
+    def formula(person, period, parameters):
         # Qualifying child under IRC 152(c), which defines for the EITC
         qualifying_child = person("is_eitc_qualifying_child", period)
-        total_eligible = tax_unit.sum(qualifying_child)
         # Disabled dependents get an additional exemption.
         disabled = where(
```

Comment:
Avoid redefining a concept this way
```suggestion
        disabled = person("is_disabled", period)
        return qualifying_child * (1 + disabled)
```